perun-base/src/main/resources/perun-roles.yml

```diff
@@ -4272,36 +4274,48 @@ perun_policies:
       - SECURITYADMIN: SecurityTeam
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: User
 
   addAdmin_SecurityTeam_Group_policy:
     policy_roles:
       - SECURITYADMIN: SecurityTeam
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Group
 
   removeAdmin_SecurityTeam_User_policy:
     policy_roles:
       - SECURITYADMIN: SecurityTeam
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: User
 
   removeAdmin_SecurityTeam_Group_policy:
     policy_roles:
       - SECURITYADMIN: SecurityTeam
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Group
```

Comment:
All the remove/add admin policies can be deleted - we now use the `setRole` instead of the regular authorization method (check other PRs adding MFA rules)

Reply:
okay, I think we can remove mentioned methods. As for creating/updating, I did not use any MFA rule because I expected that no attributes were set during those operations. Perhaps, we should discuss this, too.

Reply:
> Kritické operácie budú pre začiatok všetky politiky, ktoré sú používané v metódach meniacich stav v Perunovi.
Quote from the task description. 
This is why I believe that any method that changes the state of Perun (whether it be creating/deleting/updating any object) is supposed to be critical (and MFA covered). However, I do see where you're coming from, let's discuss it tomorrow at daily, hopefully with @balcirakpeter present.